tempesta_fw/cache.c

```diff
@@ -460,15 +458,49 @@ tfw_cache_copy_resp(TfwCacheEntry *ce, TfwHttpResp *resp, TfwHttpReq *req,
 	return 0;
 }
 
+static size_t
+__cache_entry_size(TfwHttpResp *resp, TfwHttpReq *req)
+{
+	long n;
+	size_t size = sizeof(TfwCacheEntry) - offsetof(TfwCacheEntry, ce_body);
```

Comment:
I would say that this (and the above occurrence of the same statement) should be replaced with a macro or an inline function (I prefer the latter, but it's just me).